Xamarin.Forms.Platform.Android/FastRenderers/FrameRenderer.cs

```diff
@@ -1,56 +1,35 @@
 using System;
-using System.Collections.ObjectModel;
-using System.Collections.Specialized;
 using System.ComponentModel;
 using Android.Content;
-using Android.Support.V4.View;
 using Android.Support.V7.Widget;
 using Android.Views;
 using AColor = Android.Graphics.Color;
 using AView = Android.Views.View;
 
-namespace Xamarin.Forms.Platform.Android.AppCompat
+namespace Xamarin.Forms.Platform.Android.FastRenderers
 {
-	public class FrameRenderer : CardView, IVisualElementRenderer, AView.IOnClickListener, AView.IOnTouchListener
+	public class FrameRenderer : CardView, IVisualElementRenderer, IEffectControlProvider
 	{
-		readonly Lazy<GestureDetector> _gestureDetector;
-		readonly PanGestureHandler _panGestureHandler;
-		readonly PinchGestureHandler _pinchGestureHandler;
-		readonly Lazy<ScaleGestureDetector> _scaleDetector;
-		readonly TapGestureHandler _tapGestureHandler;
-		readonly MotionEventHelper _motionEventHelper = new MotionEventHelper();
-
 		float _defaultElevation = -1f;
 		float _defaultCornerRadius = -1f;
 		int? _defaultLabelFor;
-
-		bool _clickable;
 		bool _disposed;
 		Frame _element;
-		InnerGestureListener _gestureListener;
 		VisualElementPackager _visualElementPackager;
 		VisualElementTracker _visualElementTracker;
-		NotifyCollectionChangedEventHandler _collectionChangeHandler;
+		readonly GestureManager _gestureManager;
+		readonly EffectControlProvider _effectControlProvider;
 
+		public event EventHandler<VisualElementChangedEventArgs> ElementChanged;
+		public event EventHandler<PropertyChangedEventArgs> ElementPropertyChanged;
+#pragma warning disable 414
```

Comment:
@hartez do we need this here ? i added the #praga so we can make it build .. but maybe we should just remove it..